### PR TITLE
GA SDK for azure databox

### PR DESCRIPTION
**Pull request by sojain**
## Description
Stable ga sdk for azure databox, removing unused apis, using final contracts as per the new rest api specs and arm guidelines. 
Addition of new fields, correctly marking readme and required fields
Descriptions for enums

Azure rest api spec sign off
https://github.com/Azure/azure-rest-api-specs/pull/3885
---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/DataBox/DataBox.Tests/DataBox.Tests.csproj [hunk 1]
```diff
@@ -5,7 +5,9 @@
     <Description>DataBox.Tests class library</Description>
     <AssemblyName>DataBox.Tests</AssemblyName>
     <VersionPrefix>1.0.0</VersionPrefix>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```
Comment (dsgouda) [inline src/SDKs/DataBox/DataBox.Tests/DataBox.Tests.csproj]: Why is this being modified? Please undo these changes and address any warnings present

File: src/SDKs/DataBox/Management.DataBox/Microsoft.Azure.Management.DataBox.csproj [hunk 2]
```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.DataBox</PackageId>
 		<Description>Microsoft Azure Management DataBox library</Description>
 		<AssemblyName>Microsoft.Azure.Management.DataBox</AssemblyName>
-		<Version>0.9.0-preview</Version>
+		<Version>1.0.0</Version>
 		<PackageTags>AzureDataBox;DataBox;</PackageTags>
 	</PropertyGroup>
```
Comment (dsgouda) [inline src/SDKs/DataBox/Management.DataBox/Microsoft.Azure.Management.DataBox.csproj]: Please add `PackageReleaseNotes` here
An example [here](https://github.com/Azure/azure-sdk-for-net/blob/53b16a9d7954076f9002013846c4c2840cebf92b/src/AzureStack/Admin/NetworkAdmin/Network.Admin/Microsoft.AzureStack.Management.Network.Admin.csproj#L15)

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good for the most part

File: src/SDKs/DataBox/DataBox.Tests/DataBox.Tests.csproj (hunk 1 above)
Comment (sojain) [reply src/SDKs/DataBox/DataBox.Tests/DataBox.Tests.csproj]: Not sure how this automatically creaped in, taken the fix

File: src/SDKs/DataBox/Management.DataBox/Microsoft.Azure.Management.DataBox.csproj (hunk 2 above)
Comment (sojain) [reply src/SDKs/DataBox/Management.DataBox/Microsoft.Azure.Management.DataBox.csproj]: thanks for the refered example have taken the fix

**Comment by sojain** [comment]
@dsgouda , as per your recommendation I have taken the fixes, can you take a relook. The service changes for the same have already been deployed. If things look ok can you please merge it. We have to release it on ignite 24th September as part of our GA announcement

File: src/SDKs/DataBox/DataBox.Tests/DataBox.Tests.csproj [hunk 3]
```diff
@@ -7,6 +7,9 @@
     <VersionPrefix>1.0.0</VersionPrefix>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
+  <PropertyGroup>
```
Comment (dsgouda) [inline src/SDKs/DataBox/DataBox.Tests/DataBox.Tests.csproj]: Please undo any changes to this files

**Review by dsgouda — CHANGES_REQUESTED** [review]
One of the comments was not addressed. Please fix the test csproj and this is good to merge

File: src/SDKs/DataBox/DataBox.Tests/DataBox.Tests.csproj (hunk 3 above)
Comment (sojain) [reply src/SDKs/DataBox/DataBox.Tests/DataBox.Tests.csproj]: done

**Review by dsgouda — APPROVED** [review]
LGTM will merge when CIs pass
